src/platform/routes/StudentPage/index.module.scss

```diff
@@ -0,0 +1,49 @@
+@import "~assets/styles/_global";
+
+.studentPageContainer {
+  padding: 20px 30px;
```

Comment:
increase padding from bottom

Reply:
same as the last comment of the home page the problem was not in the screen it self it was on the component called in it